crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -281,12 +281,11 @@ impl SimpleFaucet {
         amounts: &[u64],
         recipient: SuiAddress,
         uuid: Uuid,
-    ) -> Result<(TransactionDigest, Vec<ObjectID>, Vec<u64>), FaucetError> {
+    ) -> Result<(TransactionDigest, Vec<ObjectID>), FaucetError> {
         let number_of_coins = amounts.len();
         let total_amount: u64 = amounts.iter().sum();
 
         let gas_coin_response = self.prepare_gas_coin(total_amount, uuid).await;
-
```

Comment:
You know, I think we might spend the rest of each others' lifes putting in and taking out spaces in each others' code 🥲